Cargo.toml

```diff
@@ -109,7 +109,6 @@ members = [
     "crates/sui-simulator",
     "crates/sui-snapshot",
     "crates/sui-source-validation",
-    "crates/sui-source-validation-service",
```

Comment:
Is this intentional?  This is a new crate that @rvantonder is working on.

Reply:
Interesting i didn't intend to remove that...I wonder what happened. One thing that i did notice is that that crate uses `actix` @rvantonder can you please remove the use of the actix crate? We already use `axum` for web services and should not be bringing in yet another web framework for one service.